### PR TITLE
Making of project peacock

### PR DESCRIPTION
**Pull request by zdne**
DO NOT MERGE

### CODE REVIEW

File: _posts/2016-01-20-Making-of-Peacock.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: Making of Peacock
+excerpt: Behind the scenes of making Swagger support in Apiary.
+layout: post
+date: 2016-01-20 00:00:00 +0100
+author: zdenek
+published: true
+---
+
+Recently, we have launched the biggest feature in the history of Apiary: [Swagger Support](http://blog.apiary.io/2016/01/18/We-ve-got-Swagger). This project was nicknamed "The Project Peacock".
```
Comment (smizell) [inline _posts/2016-01-20-Making-of-Peacock.md]: I think it should be either:

- Project Peacock (no "the")
- The Peacock Project

I would vote for the first.

File: _posts/2016-01-20-Making-of-Peacock.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+It took us almost three months to implement it and prepare for the launch.
```
Comment (smizell) [inline _posts/2016-01-20-Making-of-Peacock.md]: So, I wonder here if you could hint that we have been laying the groundwork for Peacock for a long time.

File: _posts/2016-01-20-Making-of-Peacock.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+I would like to share with you few moments of the brave dogs, man and women
```
Comment (smizell) [inline _posts/2016-01-20-Making-of-Peacock.md]: "dogs, men and women"

Men instead of man

File: _posts/2016-01-20-Making-of-Peacock.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+---
+title: Making of Peacock
+excerpt: Behind the scenes of making Swagger support in Apiary.
```
Comment (smizell) [inline _posts/2016-01-20-Making-of-Peacock.md]: I would say "adding Swagger support"

File: _posts/2016-01-20-Making-of-Peacock.md (hunk 2 above)
Comment (zdne) [inline _posts/2016-01-20-Making-of-Peacock.md]: :+1:

**Comment by Baggz** [comment]
Great idea! Looking forward to have this published! :+1:

**Comment by zdne** [comment]
@Baggz release dat!

**Pull request merged**